tests/observability/runbook_url/utils.py

```
@@ -0,0 +1,53 @@
+
+    Args:
+        blob_url: A GitHub blob URL (e.g., https://github.com/openshift/runbooks/blob/master/alerts/.../Alert.md).
+
+    Returns:
+        The corresponding raw content URL.
+
+    Raises:
+        ValueError: If the URL does not match the expected openshift/runbooks blob format.
+    """
+    if not blob_url.startswith(RUNBOOK_BLOB_PREFIX):
+        raise ValueError(f"URL does not match expected prefix '{RUNBOOK_BLOB_PREFIX}': {blob_url}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)